src/static/LICENSE.KILTprotocol.txt

```diff
@@ -1,11 +1 @@
-This project uses npm modules @kiltprotocol/*, licensed under BSD-4-Clause.
-Their source code can be obtained from https://github.com/KILTprotocol/sdk-js.
-The license text follows.
-
-Copyright (c) 2019-2024, BOTLabs GmbH
-All rights reserved. Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
-1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
-2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-3. All advertising materials mentioning features or use of this software must display the following acknowledgement: This product includes software developed by the BOTLabs GmbH.
-4. Neither the name of the BOTLabs GmbH nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
-THIS SOFTWARE IS PROVIDED BY BOTLABS GMBH  ''AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL BOTLABS GMBH BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+This product includes software owned by KILT Foundation, namely the KILT SDK (https://www.kilt.io/developers/kilt-sdk/) and in this regards “Copyright (c) 2025, KILT Foundation“ is applicable.
```

Comment:
We didn't really have a new text for this, hope this update makes sense (I got this snippet from the text for sporran.org)

Reply:
Yes, I will speak with Svet tomorrow to get a formal update.

Reply:
what about this one? can I merge or do we want to change this?